### ISSUE TITLE
xcat run success but don't have any result

### ISSUE THREAD

**Issue by gsdt** [orf/xcat#24]
I run command `xcat --method=GET http://localhost:8080 title=Foundation title "1 results found" run retrieve`
But the xcat always return:
```
Usage:
    xcat <url> <target_parameter> [<parameters>]... (--true-string=<string> | --true-code=<code>) [--method=<method>]
         [--fast] [--oob-ip=<ip> (--oob-port=<port>)] [--stats] [--concurrency=<val>]
         [--features] [--body] [--cookie=<cookie>] [(--shell | --shellcmd=<cmd>)]
    xcat detectip
```

Please help me to fix it.

**Comment by orf** [comment]
I'm aware of this and I'm working on it. I've redesigned the internals, current master is working-ish but still needs work (OOB and shell).

**Comment by orf** [comment]
Hello,
I've just released `xcat` 1.0 on pypi 🎉 . I've redesigned the CLI and it now all works. Please see the updated README for information.

I'm working on the documentation now. Please open an issue if you run into trouble.
